cmd/ghalistener/config/config.go

```diff
@@ -18,7 +18,7 @@ import (
 
 type Config struct {
 	ConfigureUrl                string                  `json:"configure_url"`
-	AppID                       int64                   `json:"app_id"`
```

Comment:
We should probably clarify in the values.yaml file that this field can be either the app id or client id